src/uri.rs

```diff
@@ -1765,7 +1870,7 @@ test_parse! {
     [],
 
     scheme = Some("http"),
-    authority = Some("127.0.0.1"),
+    authority_part = Some(&"127.0.0.1".parse().unwrap()),
```

Comment:
All of these seem unfortunate. Can we adjust the macro to allow us to still just include a string in the test cases?

Reply:
I didn't see an obvious way to adjust the macro.